packages/internal/bridge/sdk/src/types/index.ts

```diff
@@ -274,6 +274,16 @@ export interface TxStatusResponse {
   transactions: Array<TxStatusResponseItem>
 }
 
+/**
+ * @typedef {Object} TxStatusResponse
```

Comment:
Should this be:

`* @typedef {Object} AxelarStatusResponse`

Reply:
yes! thanks